### PR TITLE
docs(pt): add `tutorials/creating-a-nuxt-module.md` translated

### PR DESCRIPTION
**Pull request by nazarepiedady**
Hello fellows, in this pull request, I am sending the translation of
the article `creating a nuxt module`, the rest will come and also
there will be updates for the already translated content.

I am counting on you @smarroufin and @ChristopheCVB.

### CODE REVIEW

**Review by smarroufin — CHANGES_REQUESTED** [review]
The rest looks good, waiting for Christophe for protuguese approval

File: content/pt/tutorials/0.creating-a-nuxt-module.md
Comment (nazarepiedady) [reply content/pt/tutorials/0.creating-a-nuxt-module.md]: @smarroufin the updates were already made.

Now the only thing that rest is the review of our
colleague @ChristopheCVB.

**Review by smarroufin — APPROVED** [review]

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**
